.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+    ecmaVersion: "2019",
+    sourceType: "module",
+  },
+  plugins: ["header"],
+  ignorePatterns: ["**/*.js", "*.d.ts"],
+  rules: {
+    "@typescript-eslint/naming-convention": "off",
+    "no-empty": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "header/header": [
+      "warn",
+      "line",
+      [" Copyright (c) Microsoft Corporation."],
```

Comment:
Let's add [the MIT header](https://docs.opensource.microsoft.com/releasing/general-guidance/copyright-headers/#typescript) back to the linter and check that it's in all the files you added.

Reply:
Changing.